guides/plugins/plugins/framework/message-queue/add-message-handler.md

```diff
@@ -172,7 +187,7 @@ shopware:
     admin_worker:
         enable_admin_worker: true
         poll_interval: 30
-        transports: ["async"]
+        transports: ["async", "async_low_priority"]
```

Comment:
```suggestion
        transports: ["async", "failed", "async_low_priority"]
```

(Only if the admin worker is supposed to consume the `failed` transport, see other comment.)

Reply:
Nope appearantly it's not